depparse/pom.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>illinois-cogcomp-nlp</artifactId>
+        <groupId>edu.illinois.cs.cogcomp</groupId>
+        <version>3.0.97</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>illinois-depparse</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>edu.illinois.cs.cogcomp</groupId>
+            <artifactId>illinois-core-utilities</artifactId>
+            <version>3.0.97</version>
+        </dependency>
+        <dependency>
+            <groupId>edu.illinois.cs.cogcomp</groupId>
+            <artifactId>illinois-edison</artifactId>
+            <version>3.0.97</version>
+        </dependency>
+        <dependency>
+            <groupId>edu.illinois.cs.cogcomp</groupId>
+            <artifactId>illinois-lemmatizer</artifactId>
+            <version>3.0.97</version>
+        </dependency>
+        <dependency>
+            <groupId>edu.illinois.cs.cogcomp</groupId>
+            <artifactId>illinois-pos</artifactId>
+            <version>3.0.97</version>
+        </dependency>
+        <dependency>
+            <groupId>edu.illinois.cs.cogcomp</groupId>
+            <artifactId>illinois-chunker</artifactId>
+            <version>3.0.97</version>
+        </dependency>
+
+        <dependency>
+            <groupId>edu.illinois.cs.cogcomp</groupId>
+            <artifactId>illinois-sl</artifactId>
+            <version>1.3.6</version>
+        </dependency>
+
+        <!-- Non-Illinois packages -->
+        <dependency>
+            <groupId>edu.cmu.cs.ark</groupId>
+            <artifactId>ChuLiuEdmonds</artifactId>
+            <version>1.0-SNAPSHOT</version>
+        </dependency>
+        <!-- The command line argument parser for MainClass.java -->
+        <dependency>
+            <groupId>net.sourceforge.argparse4j</groupId>
+            <artifactId>argparse4j</artifactId>
+            <version>0.7.0</version>
+        </dependency>
+
+        <!-- Pre-trained parser model -->
+        <dependency>
+            <groupId>edu.illinois.cs.cogcomp</groupId>
+            <artifactId>illinois-depparse-model</artifactId>
+            <version>3.0</version>
```

Comment:
How much data are you training on? Also how many features does the model have?
I trained the older version (with only basic features from Ryan's Thesis) on WSJ (~40k sentences) and it took <4 hours, if I remember correctly. 
I suspect its probably because of too many features. You can try turning on feature hashing (using the config file), but that will help only slightly  I feel.